tests/test_lexer.py

```diff
@@ -1,3 +1,8 @@
+# coding: utf-8
```

Comment:
nit: per #8 we shouldn't need to do this anymore as this coding comment is only significant for py2.

Reply:
Addressed in 13d3d479a321ce797c1fefd1472dd76ea18a2bb6.